packages/slice-machine/components/Button/index.tsx

```diff
@@ -4,23 +4,32 @@ import { Button as ThemeUIButton, Spinner } from "theme-ui";
 import { ThemeUIStyleObject } from "@theme-ui/css";
 
 export type SliceMachineButtonProps = {
+  form?: string;
+  type?: "submit" | "reset" | "button" | undefined;
   isLoading?: boolean;
+  disabled?: boolean;
   onClick?: React.MouseEventHandler<HTMLButtonElement>;
-  sx: ThemeUIStyleObject;
+  sx?: ThemeUIStyleObject;
 };
 
 const Button: React.FunctionComponent<SliceMachineButtonProps> = ({
   isLoading = false,
   children,
   onClick,
-  sx,
+  form,
+  type,
+  disabled,
+  sx = {},
 }) => {
   return (
     <ThemeUIButton
       sx={{
         alignSelf: "flex-start",
         ...sx,
       }}
+      form={form}
```

Comment:
I prefer loosing 5s one time on writing every props int the component instead of loosing 10s guessing what props is given to the component every time I read it :p